alf/algorithms/actor_critic_loss.py

```diff
@@ -151,9 +151,8 @@ def _summary():
                 td_loss=td_loss, pg_loss=pg_loss, neg_entropy=entropy_loss))
 
     def _pg_loss(self, training_info, advantages):
-        action_log_prob = tfa_common.log_probability(
-            training_info.info.action_distribution, training_info.action,
-            self._action_spec)
+        action_log_prob = dist_utils.compute_log_probability(
+            training_info.info.action_distribution, training_info.action)
         return -advantages * action_log_prob
 
     def _calc_returns_and_advantages(self, training_info, value):
```

Comment:
this function also needs to be implemented

Reply:
supporting functions implemented